scvi/core/models/__init__.py

```diff
@@ -660,10 +660,14 @@ def _validate_anndata(
         if adata is None:
             adata = self.adata
         if adata.is_view:
-            logger.warning("Input anndata is a view.")
             if copy_if_view:
-                logger.info("Making copy of anndata.")
+                logger.info("Received view of anndata, making copy.")
                 adata = adata.copy()
+            else:
+                raise ValueError(
+                    "Please run `adata = adata.copy()` or set copy_if_view to be True."
```

Comment:
```suggestion
                    "Please run `adata = adata.copy()`"
```
I think this is more catered toward users who will not have this option (copy_if_view)